solidity/random-beacon/contracts/libraries/Relay.sol

```diff
@@ -256,7 +272,33 @@ library Relay {
 
         return
             isRequestInProgress(self) &&
-            block.number > self.currentRequest.startBlock + _relayEntryTimeout;
+            block.number > self.currentRequestStartBlock + _relayEntryTimeout;
+    }
+
+    /// @notice Returns whether a relay entry request is currently in progress.
+    /// @return True if there is a request in progress. False otherwise.
+    function _isRequestInProgress(Data memory self)
```

Comment:
Can we put it next to `function isRequestInProgress(Data storage self)` version to make it clear we have two and the difference is `storage` vs `memory`?